### PR TITLE
Content index welcome page

### PR DESCRIPTION
**Pull request by vladfreel**
https://github.com/Capitalsav/socket_chat/issues/45

### CODE REVIEW

File: app/javascript/style/welcome_index.scss [hunk 1]
```diff
@@ -0,0 +1,185 @@
+.welcome_main_index{
+  width: 100%;
+  height: 100%;
+}
+.page-home {
```
Comment (polyakovigor) [inline app/javascript/style/welcome_index.scss]: Add an empty line after every block

File: app/views/welcome_pages/index_welcome.html.haml [hunk 2]
```diff
@@ -1 +1,123 @@
...
+        %h1.o-hero__header__headline
+          Where Work
+          %br
+            Happens
+        %p.o-hero__header__copy.u-margin-left--flush
+          When your team needs to kick off a project,
+          hire a new employee, deploy some code,
+          review a sales contract, finalize next year's budget,
+          measure an A/B test, plan your next office opening,
+          and more, Slack has you covered.
```
Comment (capitalsav) [inline app/views/welcome_pages/index_welcome.html.haml]: Please replace Slack with Socket Chat

File: app/views/welcome_pages/index_welcome.html.haml [hunk 3]
```diff
@@ -1 +1,123 @@
...
+        %button.btn.btn-primary.btn-lg
+          GET STARTED
+        %p.u-margin-bottom--flush.u-margin-top--medium.display-as-mini
+          Already using Slack?
```
Comment (capitalsav) [inline app/views/welcome_pages/index_welcome.html.haml]: Please replace Slack with Socket Chat

File: app/views/welcome_pages/index_welcome.html.haml [hunk 4]
```diff
@@ -1 +1,123 @@
...
+          = link_to 'Sign in.', new_user_session_path
+%section.o-section.v--lighter-gray
+  %h2.u-margin-top--flush
+    You’re in good company
+  %p.u-margin-bottom--x-large
+    Millions of people around the world have already made Slack the place where their work happens.
```
Comment (capitalsav) [inline app/views/welcome_pages/index_welcome.html.haml]: Please replace Slack with Socket Chat

File: app/views/welcome_pages/index_welcome.html.haml [hunk 5]
```diff
@@ -1 +1,123 @@
...
+      .col-sm-4.c-logowall__item.c-logowall__item_in_bottom
+        = image_tag(asset_pack_path("images/oracle.png"))
+      .col-sm-4.c-logowall__item.c-logowall__item_in_bottom.c-logowall__item_without_line
+        = image_tag(asset_pack_path("images/ticketmaster.png"))
+.d-flex.justify-content-around.try_it_for_free.padding-index
+  %div
+    %h3.c-cta--signup__headline
+      Try it for free
+    %p.c-cta--signup__content
+      Already using Slack?
```
Comment (capitalsav) [inline app/views/welcome_pages/index_welcome.html.haml]: Please replace Slack with Socket Chat

File: app/views/welcome_pages/index_welcome.html.haml [hunk 6]
```diff
@@ -1 +1,123 @@
...
+            %a{:href => "#"}  Press
+          %li.c-nav--footer__listitem
+            %a{:href => "#"}  Brand Guidelines
+
+    .col-sm-2
+      .c-nav--footer__block
+        %p.c-nav--footer__listheader Product
+        %ul.c-nav--footer__list
+          %li.c-nav--footer__listitem
+            %a{:href => "#"} Why Slack?
```
Comment (capitalsav) [inline app/views/welcome_pages/index_welcome.html.haml]: Please replace Slack with Socket Chat

File: app/views/welcome_pages/index_welcome.html.haml [hunk 7]
```diff
@@ -1 +1,123 @@
...
+          %li.c-nav--footer__listitem
+            %a{:href => "#"} eBooks
+    .col-sm-2
+      .c-nav--footer__block
+        %p.c-nav--footer__listheader Extras
+        %ul.c-nav--footer__list
+          %li.c-nav--footer__listitem
+            %a{:href => "#"} Podcast
+          %li.c-nav--footer__listitem
+            %a{:href => "#"} Slack Shop
```
Comment (capitalsav) [inline app/views/welcome_pages/index_welcome.html.haml]: Please replace Slack with Socket Chat

File: app/views/welcome_pages/index_welcome.html.haml [hunk 8]
```diff
@@ -1 +1,123 @@
...
+          %li.c-nav--footer__listitem
+            %a{:href => "#"} Slack at Work
```
Comment (capitalsav) [inline app/views/welcome_pages/index_welcome.html.haml]: Please replace Slack with Socket Chat

File: app/views/welcome_pages/index_welcome.html.haml [hunk 9]
```diff
@@ -1 +1,123 @@
...
+          %li.c-nav--footer__listitem
+            %a{:href => "#"} Slack Fund
```
Comment (capitalsav) [inline app/views/welcome_pages/index_welcome.html.haml]: Please replace Slack with Socket Chat

File: app/javascript/style/welcome_index.scss [hunk 10]
```diff
@@ -0,0 +1,185 @@
+.welcome_main_index{
+  width: 100%;
+  height: 100%;
+}
```
Comment (capitalsav) [inline app/javascript/style/welcome_index.scss:5]: Please add space between styles.

File: app/javascript/style/welcome_index.scss [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+.page-home {
+  padding-top: 9rem;
+  padding-bottom: 4rem;
+  position: relative;
+}
+h1{
```
Comment (capitalsav) [inline app/javascript/style/welcome_index.scss]: and here

**Review by polyakovigor — CHANGES_REQUESTED** [review]

File: app/javascript/style/welcome_index.scss [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+  font-family: Slack-Tiempos,"Times New Roman",Georgia,serif;
+  line-height: 1.05;
+  letter-spacing: normal;
+  margin-bottom: 2rem;
+  margin-top: 0;
+  margin-left: 0;
+  font-weight: bolder;
+  color: #323648;
+}
+.o-hero__header__copy {
```
Comment (capitalsav) [inline app/javascript/style/welcome_index.scss]: and here and all others in this file

**Review by capitalsav — COMMENTED** [review]
Please resolve issues, which I commented.

**Review by polyakovigor — APPROVED** [review]

**Review by capitalsav — APPROVED** [review]

**Pull request merged**